app/scripts/analysis/tools/powercalc/PowerCalcVizDir.directive.coffee

```diff
@@ -64,32 +65,32 @@ module.exports = class PowercalcVizDiv extends BaseDirective
         .y (d) -> yScale(d.y)
         .interpolate('basis')
 
-        for datum in data
-          _graph.append('svg:path')
-            .attr('d', lineGen(datum))
-            .data([datum])
-            .attr('stroke', 'black')
-            .attr('stroke-width', 5)
-            .attr('fill', 'blue')
-            .style('opacity', 0.5)
-
-#        # data1
-#        path1 = _graph.append('svg:path')
-#        .attr('d', lineGen(gaussianCurveData1))
-#        .data([gaussianCurveData1])
-#        .attr('stroke', 'black')
-#        .attr('stroke-width', 5)
-#        .attr('fill', 'blue')
-#        .style('opacity', 0.5)
-#
-#        # data2
-#        path2 = _graph.append('svg:path')
-#        .attr('d', lineGen(gaussianCurveData2))
-#        .data([gaussianCurveData2])
-#        .attr('stroke', 'red')
-#        .attr('stroke-width', 5)
-#        .attr('fill', 'chocolate')
-#        .style('opacity', 0.5)
+        # for datum in data
+        #   _graph.append('svg:path')
+        #     .attr('d', lineGen(datum))
+        #     .data([datum])
+        #     .attr('stroke', 'black')
+        #     .attr('stroke-width', 5)
+        #     .attr('fill', 'blue')
+        #     .style('opacity', 0.5)
+
+        # data1
+        path1 = _graph.append('svg:path')
```

Comment:
what is the reason you're manually parsing `data` object instead of doing it in a loop? the version with for loop is more general and can be later re-used for charts with more than 2 curves.